src/ui/templates/components/status-tag.njk

```diff
@@ -4,29 +4,25 @@
 
   {% set status = params.status %}
   {% set href = params.href %}
+  {% set dataCy = params.dataCy %}
  
   {% set classes %}
   {% endset %}
 
-  {% if status  === "Cannot start yet" %}
+  {% if status === "Cannot start yet" or status === "Not started yet" %}
```

Comment:
Not sure if we can in nujucks, but can we use the PAGE_VARIABLES for this incase we change any task status?

Reply:
I agree, let's do this later